corehq/apps/hqwebapp/static/hqwebapp/js/hqModules.js

```diff
@@ -0,0 +1,24 @@
+var COMMCAREHQ_MODULES = {};
+
+function hqDefine(path, moduleObject) {
```

Comment:
Thanks @millerdev that's a good idea. I think it's more in line javascript conventions anyway, so I'll change it to be a function.